tests/acceptance/bootstrap/SpacesContext.php

```diff
@@ -1345,6 +1345,33 @@ public function theUserUploadsAFileToSpace(
 		$this->featureContext->setResponse($response);
 	}
 
+	/**
+	 * @When user :user updates the content of federated share :share with :content using the WebDAV API
+	 *
+	 * @param string $user
+	 * @param string $share
+	 * @param string $content
+	 *
+	 * @return void
+	 * @throws GuzzleException
+	 * @throws Exception
+	 */
+	public function userUpdatesTheContentOfFederatedShareWithUsingTheWebdavApi(
+		string $user,
+		string $share,
+		string $content,
+	): void {
+		$spaceId = $this->featureContext->spacesContext->getSharesRemoteItemId($user, $share);
+		$this->featureContext->setResponse(
+			$this->featureContext->uploadFileWithContent(
+				$user,
+				$content,
+				$share,
+				$spaceId
+			)
```

Comment:
what happens with the share param in this function? is it used in the request?

Reply:
this should not be triggered, right? because we send some value to `$spaceId`

Reply:
I see this url being hit:
`https://localhost:10200/remote.php/dav/spaces/89f37a33-858b-45fa-8890-a1f2b27d90e1$f1c7b05d-38c4-4822-b30c-7c484b8ac287!L3RleHRmaWxlLnR4dA==/textfile.txt`
but this should work
`https://localhost:10200/remote.php/dav/spaces/89f37a33-858b-45fa-8890-a1f2b27d90e1$f1c7b05d-38c4-4822-b30c-7c484b8ac287!L3RleHRmaWxlLnR4dA==/`

you can send the empty string in that param